src/associates/src/saa-scss/saa-associates.scss

```diff
@@ -0,0 +1,90 @@
+//@import '../../../scss/config/index.scss';
+
+header,
+.newMembersToggle {
+  margin-left: 0;
+}
+
+.newMembersToggle {
+  input {
+    height: 18px;
+    width: 18px;
+    border: 2px solid #919294;
+    border-radius: 3px;
+    clip-path: unset;
+    clip: unset;
+    position: unset;
+
+    &:hover,
+    &:focus {
+      cursor: pointer;
+    }
+
+    &[type="checkbox"]:checked {
+      color: #8c1515;
+    }
+  }
+
+}
+
+.nav-tabs {
+  .nav-item {
+    background: #4d4f53;
+    color: #fff;
+
+    &.active {
+      background: #8c1515;
+    }
+
+    @media only screen and (min-width: 768px) {
+      padding: 0.8rem 1.5rem;
+    }
+  }
+}
+
+.react-autosuggest__container {
+  position: relative;
+}
+
+.react-autosuggest__input {
+  padding: 0.6rem 2rem 0.8rem;
+  width: 50%;
+}
+
+.react-autosuggest__suggestions-container--open {
+  width: 50%;
+  height: 250px;
```

Comment:
This is missing `max-width: 46rem` that the input has. If you add this, then the dropdown width will match the input width and be consistent.